tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKCorePlugin.java

```diff
@@ -221,6 +226,41 @@ public void sdksRemoved( SDK[] sdks )
         };
 
         SDKManager.getInstance().addSDKListener( this.sdkListener );
+
+        CoreUtil.getWorkspace().addResourceChangeListener( new IResourceChangeListener()
+        {
+
+            @Override
+            public void resourceChanged( IResourceChangeEvent event )
+            {
+                try
+                {
+                    if( event.getType() == IResourceChangeEvent.PRE_DELETE )
+                    {
+                        IProject project = (IProject) event.getResource();
+
+                        if( SDKUtil.isSDKProject( project ) )
+                        {
+                            String parentName = project.getLocation().toFile().getParentFile().getName();
+                            IProject sdkProject = SDKUtil.getWorkspaceSDKProject();
+                            IFolder parentFolder = sdkProject.getFolder( parentName );
+
+                            CoreUtil.deleteProjectResourceFilter( parentFolder, project );
```

Comment:
deleteProjectResource Filter should be located in the same place where it was added.